index.js

```diff
@@ -1,9 +1,7 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { Image, Platform, StyleSheet, TouchableOpacity, View, ViewPropTypes } from 'react-native';
+import React, { Component, PropTypes } from 'react';
+import { View, StyleSheet, Image, TouchableOpacity, Platform, NativeModules } from 'react-native';
```

Comment:
Any specific reasons for changing to use the built-in `PropTypes` instead of the separate `prop-types` library?

Reply:
built-in PropTypes are breaking since react does not support them, we should use separate prop-types library.